UsageExample/Program.cs

```diff
@@ -109,17 +116,38 @@ private static async Task Main(string[] args)
 
     var result = blobInfos[0];
 
+    var content = File.ReadAllBytes(@"C:\\Users\\iaguiar\\Desenvolvimento\\MyProjects\\OnSSH\\ArmoniK.Extension.CSharp.Scratch\\UsageExample\\publish.zip");
```

Comment:
Change to something generic, like a relative path if your zip is created dynamically in the tests